index.js

```diff
@@ -80,34 +119,109 @@ module.exports = {
         '@rollup/plugin-babel',
         'decorator-transforms',
 
+        // Dependencies out of date from upstream
+        // (or waiting for the release train to catch up)
         'eslint@latest',
         'eslint-plugin-ember@latest',
         'eslint-plugin-n@latest',
         '@babel/eslint-parser@latest',
+        'ember-template-lint@latest',
+
+        '@ember/string@latest',
         'ember-resolver@latest',
         'ember-load-initializers@latest',
+        'qunit@latest',
+        'qunit-dom@latest',
+        'stylelint@latest',
+        'stylelint-prettier@latest',
+        'stylelint-config-standard@latest',
```

Comment:
this might be a bit too "latest", it opens this up to float past what we know works. If you want to update it to something past what is actually in the `ember-cli@latest` right now then put a `^` with a major in it

Reply:
Turns out ember cli is already using latest for stylelint, so i could remove some of these now with the latest ember-cli release

Reply:
yes that would be great 👍 

for the record ember-cli is never using `latest`, it always updates the packages that it's using but defines a version. This `@latest` here is a lot more dangerous a change. For most of the other things in this list it's harmless but stylelint has bit us in the past and it would be a shame to have the app blueprint broken for a lint library 🙈

Reply:
Ye, i just meant that until 5.12, ember-cli was several majors behind on a lot of things, leading to situations like this: 

https://discord.com/channels/480462759797063690/480501885837770763/1294265500569960510